src/wallet-redux.js

```diff
@@ -220,9 +226,12 @@ export function reducer(state, action) {
                 activeWalletId: action.walletId,
             });
 
-        case TypeKeys.UPDATE_ACCOUNT_LABEL:
+        case TypeKeys.SET_CLAIMED_FLAG:
```

Comment:
Maybe the naming should include the word Cashlink as this is only relevant for cashlinks.

Reply:
This has been removed from wallet-redux.